src/quicklook.cs

```diff
@@ -170,6 +171,88 @@ interface QLPreviewItem {
 #endif
 	}
 
+	delegate bool QLPreviewReplyDrawingHandler (CGContext context, QLPreviewReply reply, out NSError error);
+	delegate NSData QLPreviewReplyDataCreationHandler (QLPreviewReply reply, out NSError error);
+	delegate CGPDFDocument QLPreviewReply_UIDocumentCreationHandler (QLPreviewReply reply, out NSError error);
+
+	[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface QLPreviewReply
+	{
+		[Export ("stringEncoding")]
+		nuint StringEncoding { get; set; }
+
+		[Export ("attachments", ArgumentSemantic.Copy)]
+		NSDictionary<NSString, QLPreviewReplyAttachment> Attachments { get; set; }
+
+		[Export ("title")]
+		string Title { get; set; }
+
+		[Export ("initWithContextSize:isBitmap:drawingBlock:")]
+		IntPtr Constructor (CGSize contextSize, bool isBitmap, QLPreviewReplyDrawingHandler drawingHandler);
+
+		[Export ("initWithFileURL:")]
+		IntPtr Constructor (NSUrl fileURL);
+
+		[Export ("initWithDataOfContentType:contentSize:dataCreationBlock:")]
+		IntPtr Constructor (UTType contentType, CGSize contentSize, QLPreviewReplyDataCreationHandler dataCreationHandler);
+
+		// QLPreviewReply_UI
+		[Export ("initForPDFWithPageSize:documentCreationBlock:")]
+		IntPtr Constructor (CGSize defaultPageSize, QLPreviewReply_UIDocumentCreationHandler documentCreationHandler);
+	}
+
+	[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface QLPreviewReplyAttachment
+	{
+		[Export ("data")]
+		NSData Data { get; }
+
+		[Export ("contentType")]
+		UTType ContentType { get; }
+
+		[Export ("initWithData:contentType:")]
+		IntPtr Constructor (NSData data, UTType contentType);
+	}
+
+	[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface QLFilePreviewRequest
+	{
+		[Export ("fileURL")]
+		NSUrl FileURL { get; }
+	}
+
+	[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface QLPreviewProvider : NSExtensionRequestHandling
+	{
+	}
+
+	[iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface QLPreviewSceneOptions
+	{
+		[Export ("initialPreviewIndex")]
+		nint InitialPreviewIndex { get; set; }
+	}
+
+	// TODO: BaseType UIWindowSceneActivationConfiguration must first be implemented in UIKit
```

Comment:
[[QuickLook] QLPreviewSceneActivationConfiguration not yet bound in Xcode13](https://github.com/xamarin/xamarin-macios/issues/12335) because base type UIWindowSceneActivationConfiguration is not yet bound in UIKit